Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+        }
+
+        public static QtestTest GetQtestTest(string testID)
+        {
+           
+            return new QtestTest();
+        }
+
+        public static ObservableList<ExternalItemFieldBase> GetALMItemFields(ALM_Common.DataContracts.ResourceType resourceType)
+        {
+            ObservableList<ExternalItemFieldBase> fields = new ObservableList<ExternalItemFieldBase>();
+
+            //if (QCRestAPIConnect.QcRestClient == null)
+            //{
+            //    string qcbin = "qcbin";
+            //    QCRestAPIConnect.QcRestClient = new QCClient(ALMCore.AlmConfig.ALMServerURL.TrimEnd(qcbin.ToCharArray()), ALMCore.AlmConfig.ALMUserName, ALMCore.AlmConfig.ALMPassword, ALMCore.AlmConfig.ALMDomain, ALMCore.AlmConfig.ALMProjectName, 12);
+            //}
+
+            //if (QCRestAPIConnect.QcRestClient.Login())
+            {
+                if (resourceType == ALM_Common.DataContracts.ResourceType.ALL)
+                    return GetALMItemFields();
+                else
+                {
+                    //string fieldInRestSyntax = QCRestAPIConnect.ConvertResourceType(resourceType);
+                    List<QTestApiModel.FieldResource> fieldsCollection = fieldApi.GetFields((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), resourceType.ToString());
+
+                    fields.Append(AddFieldsValues(fieldsCollection, resourceType.ToString()));
+                }
+            }
+
+
+            //fieldApi = new QTestApi.FieldApi(connObj.Configuration);
+            //fieldApi.GetFields((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), resourceType.ToString());
+
+            return fields;
+        }
+        private static ObservableList<ExternalItemFieldBase> GetALMItemFields()
+        {
+            ObservableList<ExternalItemFieldBase> fields = new ObservableList<ExternalItemFieldBase>();
+            fieldApi = new QTestApi.FieldApi(connObj.Configuration);
+
+            //QC   ->testSet,    testCase,  designStep,testInstance,designStep,run
+            //QTest->test-suites,test-cases,
+
+            //string testSetfieldInRestSyntax = QCRestAPIConnect.ConvertResourceType(ALM_Common.DataContracts.ResourceType.TEST_SET);
+            List<QTestApiModel.FieldResource> testSetfieldsCollection = fieldApi.GetFields((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), "test-suites");
+
+            //string testCasefieldInRestSyntax = QCRestAPIConnect.ConvertResourceType(ALM_Common.DataContracts.ResourceType.TEST_CASE);
+            List<QTestApiModel.FieldResource> testCasefieldsCollection = fieldApi.GetFields((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), "test-cases");
+
+            ////string designStepfieldInRestSyntax = QCRestAPIConnect.ConvertResourceType(ALM_Common.DataContracts.ResourceType.DESIGN_STEP);
+            //List<QTestApiModel.FieldResource> designStepfieldsCollection = fieldApi.GetFields((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), "test-suites");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)